website/src/components/CollapsableText.tsx

```diff
@@ -1,16 +1,16 @@
-import React, { ReactNode } from "react";
+import { ReactNode } from "react";
 
-export const CollapsableText = ({
-  text,
-  maxLength = 220,
-}: {
-  text: ReactNode;
-  maxLength?: number;
-  isDisabled?: boolean;
-}) => {
+export const CollapsableText = ({ text, maxLength = 220 }: { text: ReactNode; maxLength?: number }) => {
   if (typeof text !== "string" || text.length <= maxLength) {
     return <>{text}</>;
   } else {
-    return <span>{text.substring(0, maxLength - 3)}...</span>;
+    const visibleText = text.substring(0, maxLength - 3);
+    const dots = visibleText[visibleText.length - 1] === "." ? ".." : "...";
```

Comment:
There is no need for this check, it is not problem.

also, it does not handle all cases, what if your text ends with two dots? with three?

as said, not needed.

Reply:
Okay removed it.
I have a question in `SortableItem.tsx` we're using `useMemo` to change the background when the card is active. Isn't this overkill? we could use Tailwindcss `hover:` for this and for the cursor as well.

Reply:
Yes, our code needs simplification, feel free to clean up if you find anything worthy, preferably in another PR.